### ISSUE TITLE
[MHXX] Citra Crashes When someone using some weapon like Lightning Bow (Astalos Bow)

### ISSUE THREAD

**Issue by greaper310** [citra-emu/citra#4637]
when i use Lightning Bow, all my friends is crashed while i'm not... and when my friend use it, my Citra crashed while he not

it's happen on all Citra version

**Comment by wwylele** [comment]
Please upload the log, especially for the one that crashes.

**Comment by greaper310** [comment]
[citra_log.txt](https://github.com/citra-emu/citra/files/2856750/citra_log.txt)

that's it, sorry if i was too late to send it, because i need a friend to get this log

**Comment by tywald** [comment]
> GL_VERSION: 3.3.0 NVIDIA 398.26

Can you update the drivers and see if you still crash? Latest is 418.91, get from Nvidia's website or from GeForce Experience.

**Comment by wwylele** [comment]
The log doesn't really look like a GPU crash though

**Comment by greaper310** [comment]
That was one of my friend log, and of course isn't because GPU or Driver, i have a friend that have i7 6700 + GTX 1050ti with latest driver 418.91, still crash too

**Comment by tywald** [comment]
Yeah it was more of a "just in case" since no one should run with outdated drivers :P

I can try to reproduce it but I would like to know exactly what MHXX you are using. Is it just a clean dump with no updates or patches? Or with the 1.4.0 update? Or with a translation patch, if so which one? Thanks!

**Comment by greaper310** [comment]
1.4.0 Patch v5 English

equip Lightning Bow and play with your friends, after several minutes (longest is 8 minutes), their Citra will crashed while you're not

that will happen to you if your friend equip it too

**Comment by hayhalo91** [comment]
Finally someone report this, honestly i'm doubt before this is because my PC performance, but after crosscheck with all my friends, this is totally bug, even my friend with i7 8700 + 1070ti crashed too when i'm using that astalos bow, and i'm crashed too when he using astalos bow

**Comment by tywald** [comment]
Is it a specific upgrade of the bow or does it happen with all of them?

I haven't got around testing yet(busy with stuff). Using multiple Citra instances should theoretically accomplish the same as if you bring in a friend so I will test that first.

@wwylele if this would be English patch related then this is not something that would get fixed right? Unless it doesn't crash on the 3DS. I want to make sure so I'm not spreading misinformation.

**Comment by wwylele** [comment]
@tywald First we need to know whether the same bug happens with the original game.

**Comment by hayhalo91** [comment]
For my experience, it only happen to astalos bow, the other bow have no problem, but not sure for deviant astalos bow (i don't have it, so i can't check it by myself)

**Comment by DragonHunter966** [comment]
Is this real? Now i know what caused my citra crashed yesterday, my friend using astalos bow, mine version is 1.2.0

**Comment by hayhalo91** [comment]
I think this happen to all astalos weapon, but only bow that can cause crash faster, i use astalos longsword while fight amatsu and my friend crashed after 17 minutes gameplay

Another case is astalos sns, i use it twice in arena and crashed between 15 - 20 minutes

**Comment by greaper310** [comment]
report updated, this bug only happen to PC with GPU 1050ti below or CPU below i5 7400, not sure this is because CPU or GPU

**Comment by hayhalo91** [comment]
HW.Memory <Error> core/memory.cpp:Read:188: unmapped Read32 @ 0x000000A4

This is the last log when i'm crashed, so what is this error?
